### ISSUE TITLE
parsing time zone offset with second fails

### ISSUE THREAD

**Issue by palindrom615** [moment/moment-timezone#772]
```javascript
moment("1899-06-28T23:27:52.000+08:27:52") // invalid date!
```

Above code fails because `moment` cannot parse timezone offset with the second part.

The format is not standard, **it is valid Korean standard time** before April 1st, 1908. (see this: https://ko.wikipedia.org/wiki/%ED%95%9C%EA%B5%AD_%ED%91%9C%EC%A4%80%EC%8B%9C) Can this be considered? This bug affects business logic on my work.

**Comment by saschanaz** [comment]
The document only says it was `+08:28`, not `+08:27:52` though...

**Comment by palindrom615** [comment]
@saschanaz ,

> The document only says it was `+08:28`, not `+08:27:52` though...

Yeah, Wikipedia does not mention about the second unit of offset. The premodern time standard is quite an ambiguous problem, but implementations like java `ZonedDateTime` are designed with later one:
![1-934-0f55c3](https://user-images.githubusercontent.com/10936496/61841963-78d2f580-aed1-11e9-9c32-269f7f62b951.jpg)

**Comment by saschanaz** [comment]
The `Intl` implementation in Firefox also generates `+08:27:52`:

```
> Intl.DateTimeFormat("default", { timeZone: "Asia/Seoul", timeZoneName: "long" }).format(new Date("1900-01-01T00:00:00"))
"1/1/1900, GMT+08:27:52"
```

**Comment by Hoto-Cocoa** [comment]
According Wikipedia's [Time in North Korea](https://en.wikipedia.org/wiki/Time_in_North_Korea) Document, "Hanyang (Seoul) which was calculated to be **UTC+08:27:52**[Ref: A Bridge between Conceptual Frameworks: Sciences, Society and Technology Studies edited by Raffaele Pisano]. In 1442".

![image](https://user-images.githubusercontent.com/21969938/61895039-a3ae5f80-af4c-11e9-871e-e988e4a0072f.png)

**Comment by ichernev** [comment]
Fair point, but this is a bug in moment, not moment-timezone. I'll leave it here for now.
